test/controllers/messages_controller_test.rb

```diff
@@ -0,0 +1,9 @@
+require 'test_helper'
+
+class MessagesControllerTest < ActionDispatch::IntegrationTest
+  test "should get reply" do
+    get messages_reply_url
```

Comment:
have you ran this test? It kind of seems like it would fail, given the only route defined is a POST

Reply:
I've never run this. I think it was generated when I initially started the ticket and ran one of the commands from the docs.